backend/.env.example

```
@@ -1,8 +1,8 @@
 PORT=3001
-SUPABASE_URL=your_supabase_url_here
-SUPABASE_KEY=your_supabase_anon_key_here
-GEMINI_API_KEY=your_gemini_api_key_here
-FRONTEND_URL=your_front_end_url
+SUPABASE_URL=https://ghcrpnhjwwgxhfzqbkdm.supabase.co
+SUPABASE_KEY=sb_publishable_zwoP8mLKtd4eHdPMqUjJJw_gJjJ386m
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_KEY key should go before the SUPABASE_URL key

(UnorderedKey)